README.md

```diff
@@ -235,6 +237,20 @@ script as a wrapper, for example:
 less --tabs 4 -RF "$@"
 ```
 
+## Using `bat` on Windows
+
+`bat` mostly works out-of-the-box on Windows, but a few features may need extra configuration.
+
+### Paging
+
+Windows only includes a very limited pager in the form of `more`. You can download a Windows binary for `less` [from its homepage](http://www.greenwoodsoftware.com/less/download.html) or [through Chocolatey](https://chocolatey.org/packages/Less). To use it, place the binary in a directory in your `PATH` or [define an environment variable](#using-a-different-pager).
+
+### Colours
+
+Windows 10 natively supports colours in both `conhost.exe` (Command Prompt) and PowerShell since [v1511](https://en.wikipedia.org/wiki/Windows_10_version_history#Version_1511_(November_Update)), as well as in newer versions of bash. On earlier versions of Windows, you can use [Cmder](http://cmder.net/), which includes [ConEmu](https://conemu.github.io/).
+
+**Note:** The Git and MSYS versions of `less` do not correctly interpret colours on Windows. You can disable `bat`’s coloured output using `--color=never`.
```

Comment:
Hm.. isn't using `bat` without general color support kind of pointless? I think I would rather advise to use `--paging=never` (or `BAT_PAGER=""`) in this case.

Reply:
Sure, that makes sense. I’ll update it.

Reply:
> I think I would rather advise to use `--paging=never` (or `BAT_PAGER=""`) in this case.

@sharkdp AFAIK empty environment variables are not supported on windows and can neither be set via `set`/`setx` nor via GUI and only by editing the registry

Reply:
@aasedb Thank you for the hint. Can you please open a new ticket/PR so this can be properly tracked?